msal/src/main/java/com/microsoft/identity/client/BrowserTabActivity.java

```diff
@@ -53,15 +56,21 @@
  * </pre>
  */
 public final class BrowserTabActivity extends Activity {
-    //private static final String TAG = BrowserTabActivity.class.getSimpleName();
+    private static final String TAG = BrowserTabActivity.class.getSimpleName();
 
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (savedInstanceState == null
                 && getIntent() != null
                 && !StringUtil.isEmpty(getIntent().getDataString())) {
-            startActivity(BrowserAuthorizationFragment.createCustomTabResponseIntent(this, getIntent().getDataString()));
+            final Intent responseIntent = BrowserAuthorizationFragment.createCustomTabResponseIntent(this, getIntent().getDataString());
+            if (responseIntent != null) {
+                startActivity(responseIntent);
+            } else {
+                Logger.warn(TAG, "Received NULL response intent. Unable to complete authorization.");
+                Toast.makeText(getApplicationContext(), "Unable to complete authorization as there is no interactive call in progress. This can be due to closing the app while the authorization was in process.", Toast.LENGTH_LONG).show();
```

Comment:
We might not want to pop this toast as it will be visible (and potentially confusing) to end users.

Reply:
I agree with you. I'll probably remove this when it actually goes in prod, but for now this is going to be on a private build.